src/terminal/terminal.c

```diff
@@ -243,7 +243,7 @@ void guac_terminal_reset(guac_terminal* term) {
 
     /* Clear terminal */
     for (row=0; row<term->term_height; row++)
-        guac_terminal_set_columns(term, row, 0, term->term_width, &(term->default_char));
+        guac_terminal_set_columns(term, row, 0, term->term_width-1, &(term->default_char));
```

Comment:
This is wrong now, yeah? Since the flag was moved out of the buffer and into a separate field.

Reply:
Without the -1 here, row->length is larger on the initial screen than on new lines once you get to the bottom of the screen.
Everywhere else in the code initializations are done on term_width-1 (probably because the scrollbar is on top).

Reply:
In that case, would you mind adding a comment to the code explaining this?

Reply:
Comment added to the code.